### ISSUE TITLE
Missing docs on rust-lang-nursery.github.io for `write_literal`

### ISSUE THREAD

**Issue by bjgill** [rust-lang/rust-clippy#2656]
Found with Clippy 0.0.193

The new `write_literal` lint points you to a seemingly missing piece of documentation. As far as I can tell, there's nothing at https://rust-lang-nursery.github.io/rust-clippy/v0.0.193/index.html#write_literal.

```
warning: writing a literal with an empty format string
  --> src/bin/rm/main.rs:61:30
   |
61 |     write!(output, "{:>12}", "Removing")?;
   |                              ^^^^^^^^^^
   |
   = note: #[warn(write_literal)] on by default
   = help: for further information visit https://rust-lang-nursery.github.io/rust-clippy/v0.0.193/index.html#write_literal
```

(https://github.com/rust-lang-nursery/rust-clippy/issues/2634 covers the fact that this is a false positive in the first place)

**Comment by oli-obk** [comment]
Yea, rustc broke our macro checks :(

**Comment by phansch** [comment]
Looks like `write_literal` is present since v.0.0.194.
Since this issue is quite old, and most people have upgraded since then, I'm going ahead and close this issue.

The docs are present here for example: https://rust-lang.github.io/rust-clippy/master/index.html#write_literal
